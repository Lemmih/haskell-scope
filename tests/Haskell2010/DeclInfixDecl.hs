{-# LANGUAGE Haskell2010 #-}
module DeclInfixDecl where

infix +
infixr +
infixl +

infix 1 +
infixr 1 +
infixl 1 +

(+) = (+)
