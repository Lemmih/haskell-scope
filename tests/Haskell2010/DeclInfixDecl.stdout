Scope errors:

Definitions:
  Definition 1, from DeclInfixDecl.decl6:
    ([4m+[0m) = (+)

Use sites:
  Definition used: 1
    infix [4m+[0m
  Definition used: 1
    infixr [4m+[0m
  Definition used: 1
    infixl [4m+[0m
  Definition used: 1
    infix 1 [4m+[0m
  Definition used: 1
    infixr 1 [4m+[0m
  Definition used: 1
    infixl 1 [4m+[0m
  Definition used: 1
    (+) = ([4m+[0m)

Exports:
  +, Value from DeclInfixDecl
